tools/roslaunch/scripts/roslaunch-check

```diff
@@ -116,10 +114,12 @@ if __name__ == '__main__':
         print("FAILURE:\n%s"%error_msg, file=sys.stderr)
         with open(test_file, 'w') as f:
             message = "roslaunch check [%s] failed"%(roslaunch_path)
-            f.write(rosunit.junitxml.test_failure_junit_xml(test_name, message, stdout=error_msg))
+            f.write(rosunit.junitxml.test_failure_junit_xml(test_name, message, stdout=error_msg,
+                    class_name="roslaunch.RoslaunchCheck", testcase_name="%s_%s" % (pkg, outname)))
         print("wrote test file to [%s]"%test_file)
         sys.exit(1)
     else:
         print("passed")
         with open(test_file, 'w') as f:
-            f.write(rosunit.junitxml.test_success_junit_xml(test_name))
+            f.write(rosunit.junitxml.test_success_junit_xml(test_name, stdout="",
+                    class_name="roslaunch.RoslaunchCheck", testcase_name="%s_%s" % (pkg, outname)))
```

Comment:
Oh of course. I thought I had already removed it; thanks for the catch. Fixed now.